client/checkout/classic/index.js

```diff
@@ -38,8 +38,10 @@ jQuery( function ( $ ) {
 	);
 	const elements = api.getStripe().elements();
 
-	// In the future this object will be loaded with customer information through `wp_localize_script`.
-	const preparedCustomerData = {};
+	// Customer information for Pay for Oder and Save Payment method.
+	/* global wcpayCustomerData */
```

Comment:
```suggestion
	/* global wcpayCustomerData from `wp_localize_script` */
```

Reply:
The linter complains if I add the details.